local/read_api_fs_args.py

```diff
@@ -46,3 +46,16 @@ def read_n_separate_args(directory, separated_args_names=[]):
             if len(arg_value) > 0:
                  completed_cmd_args.append(arg_value)
     return completed_cmd_args, separated_completed_cmd_args
+
+def read_args_dict(directory):
```

Comment:
maybe it's good idea to reorganize this utils-modules as:
gui_api ???-> common_api_utils <-filesystem_api